ui/00_message/skill/custom_skill_info_01.xml

```diff
@@ -136,6 +136,12 @@ LV.6: Increases range. Increases counter duration.&lt;/COL&gt;</translation>
 初段を敵に接触させれば連撃に移行する
 &lt;COL ffdc78&gt;LV UP毎に攻撃威力の上昇
 LV.６でモーション変化／攻撃ヒット数増加&lt;/COL&gt;</original>
+    <translation>A high-powered sword dance that covers a wide area.
+If the first hit is successfully landed, it will become a full on
+combo attack.
+&lt;COL ffdc78&gt;LV UP: Increases damage of the skill.
+LV.６: Increases the number of hits in the attack, and changes
```

Comment:
Please change the fullwidth character to the regular character (in this case, LV.６should be LV.6). This applies to any other translated messages as well.